cumulusci/core/config/MergedConfig.py

```diff
@@ -0,0 +1,96 @@
+import io
+import yaml
+
+from builtins import bytes, int, str
+from collections import OrderedDict
+
+from cumulusci.core.config import BaseConfig
+from cumulusci.core.exceptions import ConfigMergeError, ConfigError
+
+
+class MergedConfig(BaseConfig):
+    """ A merged config takes a collection of dicts and merges them in priority order."""
```

Comment:
I don't really understand why this is its own object instead of a function used to populate the config of another BaseConfig subclass.